libcxx/utils/ci/run-buildbot

```diff
@@ -328,6 +329,8 @@ generic-cxx26)
     generate-cmake -C "${MONOREPO_ROOT}/libcxx/cmake/caches/Generic-cxx26.cmake"
     check-runtimes
     check-abi-list
+    python3 $MONOREPO_ROOT/clang-tools-extra/clang-tidy/tool/run-clang-tidy.py \
+        -clang-tidy-binary clang-tidy-20 -warnings-as-errors \* -source-filter ".*libcxx/src.*" -p "${BUILD_DIR}"
```

Comment:
We should probably run this as part of our test suite instead like we do for the headers. We can access the sources as `%{libcxx-dir}` from `lit`.